Patches/MeetingHudPatch.cs

```diff
@@ -190,8 +190,6 @@ public static bool Prefix(MeetingHud __instance)
                 var player = Utils.GetPlayerById(ps.TargetPlayerId);
                 var playerRoleClass = player.GetRoleClass();
 
-                // Hide roles vote
-                if (playerRoleClass.HideVote(ps)) continue;
```

Comment:
I think need leave this feature as some new roles may use this

Or just comment out this code so as not to lose it

Reply:
hmm okay